### DISCUSSION TITLE
请问可以支持在视频动态里看到不同up主的所有视频吗

### DISCUSSION THREAD

**Discussion by omniversexjy** [ywmoyue/biliuwp-lite#505] [feature]
### 问题描述

就像之前的官方的uwp的视频动态那样

### 描述解决方案

![image](https://github.com/ywmoyue/biliuwp-lite/assets/63655081/3d79a115-1448-477f-a4dc-8a456f760e2b)
类似于这样

### 代码贡献

否

**Comment by ywmoyue** [comment]
不懂，为什么要在动态里面看到up主的所有视频，要看关注的up主的所有视频的话可以点击右上角头像-> 关注 -> 找到up主点进去

**Reply by omniversexjy** [reply]
因为这样的话就能很方便看到自己有没有漏看某个up的哪一条新视频了😊

**Comment by ywmoyue** [comment]
且我看哔哔最新版似乎没这功能
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/287644a9-beae-4aec-8282-c2469bd3244e)

**Comment by GD-Slime** [comment]
@omniversexjy  你说的功能是否是
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/9724a44c-2e04-4dac-af98-2754c65e16fa)
视频动态？打开此Tab后就可以看到所有你关注的up主最近新发的视频。

**Comment by omniversexjy** [comment]
没有呀，我这里左边一栏可以看到关注的up主。如果他们有我没看到的更新，就会像我拍的照片一样置顶然后加上一个蓝点。应该和手机app动态里顶上那一栏是一个道理。
![IMG_6879](https://github.com/ywmoyue/biliuwp-lite/assets/63655081/29fff79e-993e-40ac-80e1-628e5d14411c)
